pi_controller/cattown/.github/workflows/release.yml

```diff
@@ -0,0 +1,79 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+      - master
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+
+      - name: Set up Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: "3.10"
```

Comment:
We'll need to confirm the latest stable OpenCV 4.9.0 works with Pyhthon 3.10, or downgrade OpenCV if that makes more sense

Reply:
Thanks! Yea I haven't tried that yet. Happy to do so before merging!